docs/FAQ.md

```diff
@@ -109,6 +109,17 @@ commit, then run `jj restore --from Y --to @-` to restore the parent commit
 to the old state, and `jj restore --from X` to restore the new working-copy
 commit to the new state.
 
+### What do I do with divergent changes?
+
+A divergent change (denoted by `??` after the change ID in the log) represents
+a change that has two or more visible commits associated with it. Most commonly,
+the way to resolve this is to abandon the unneeded commits.
```

Comment:
Here a sentence on how to resolve divergent branches would also be nice. Here's my suggesion for it, `For divergent branches, setting the branch to the correct version via "jj branch set" should resolve the divergence. As repeatedly abandoning commits does not resolve the divergence.`

Reply:
nit: drop the "As" there (or join the sentences with ", as"). More importantly, I'm not sure we should even include that last sentence. First, it's not actually true - repeatedly abandoning commits will resolve the divergence, once the one side becomes an ancestor of the other. Do you think it's a common thing to think that repeatedly abandoning commits would be a good strategy for resolving conflicted branches? If not, it seems better to not mention it at all.

Reply:
> Do you think it's a common thing to think that repeatedly abandoning commits would be a good strategy for resolving conflicted branches? If not, it seems better to not mention it at all.

I've mentioned it as a counter example, as that was my experience from the last time we talked about it in [Discord].

[Discord]: https://discord.com/channels/968932220549103686/969829516539228222/1144289598579683429

Reply:
I can see how thinking that abandoning one side would help, but I don't think users are going to think that repeatedly abandoning commits will help. So I'm fine with something like "Note that abandoning one side of the conflicted branch does not help - it just moves that side of the branch to the parent commit.".

Reply:
Perhaps this should be a separate question? We can also link to the Glossary, were I think Martin carefully explained the difference and similarity between conflicted branches and change ids.

Alternatively, we could make it a separate paragraph in this question, but probably at the end.

Reply:
Yes, I think it's makes sense to make it a separate question. It would be easier to link to that way. And it also doesn't block this PR :)

Reply:
OK, I'll add this as a separate question 👍